src/components/ItemsCarousel/ItemsCarousel.js

```diff
@@ -0,0 +1,117 @@
+import { useState } from "react";
+import Items from "./Items";
+import "./ItemsCarousel.scss";
+
+const ItemsCarousel = () => {
+  const [slideIndex, setSlideIndex] = useState(0);
+
+  const onClick = event => {
+    const isPrev = event.target.className.includes("prev");
+    const isNext = event.target.className.includes("next");
+    if (isPrev) {
+      setSlideIndex(curr => --curr);
+    }
+    if (isNext) {
+      setSlideIndex(curr => ++curr);
+    }
+  };
+
+  const firstSlide =
+    slideIndex === 0
+      ? {
+          style: { transform: "translateX(-80px)" },
+        }
+      : null;
+  const lastSlide =
+    slideIndex === 1
+      ? {
+          style: { transform: "translateX(80px)" },
+        }
+      : null;
+
+  return (
+    <div className="carouselContainer">
```

Comment:
```suggestion
    <div className="itemsCarousel">
```
* 최상단 태그는 컴포넌트 이름이랑 같게 수정 해 주세요